ONSdigital/eq-questionnaire-schemas
# Update census schemas to use address lookup answers

### What is the context of this PR?

Swaps out individual address answers for `Address` answers across all census schemas.

I have added `region_code` as a `ext-str` arg to the Jsonnet commands, in addition to it being passed as a `tla-str` arg. The [style guide](https://github.com/ONSdigital/eq-questionnaire-schemas/blob/master/style_guide.md) states this is the approach that should be used, as the variable is used in more than 5 blocks. I haven't however refactored all the other places that use the value passed via the `tla-str` arg, as that would complicate the review of this pull request. I have raised a card for refactoring to use the `ext-str` arg for `region_code` throughout.


### How to review

- Check all address answers have been changed to `Address` answers
- Check for any places an address placeholder is used

### Checklist

- [ ] Jsonnet files conform to the latest [style guide](/ONSdigital/eq-questionnaire-schemas/blob/master/style_guide.md)

### Quick Launch

#### England

- [Household](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=H&region_code=GB-ENG&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/update-census-schemas-to-use-address-lookup-answers/schemas/en/census_household_gb_eng.json)

- [Individual](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-ENG&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/update-census-schemas-to-use-address-lookup-answers/schemas/en/census_individual_gb_eng.json)

- [CCS](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-ENG&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/update-census-schemas-to-use-address-lookup-answers/schemas/en/ccs_household_gb_eng.json)

#### Northern Ireland

- [Household](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-NIR&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/update-census-schemas-to-use-address-lookup-answers/schemas/en/census_household_gb_nir.json)

- [Individual](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-NIR&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/update-census-schemas-to-use-address-lookup-answers/schemas/en/census_individual_gb_nir.json)
